### DISCUSSION TITLE
unable to understand data get from read method data is this ���

### DISCUSSION THREAD

**Discussion by waheedshabeer** [kenjdavidson/react-native-bluetooth-classic#185] [Q&A]
when i scan rfid tag with scanner i get this data from read method  ��� how to decrypt or unde
<img width="518" alt="code" src="https://user-images.githubusercontent.com/63425832/170200468-4f5cb40a-05d8-40e7-90af-499c96637385.PNG">
<img width="119" alt="read" src="https://user-images.githubusercontent.com/63425832/170200478-94ef95d3-c711-469c-a583-26a0281f6d19.PNG">
rstand this data?

**Comment by kenjdavidson** [comment]
It looks like it's encoded differently then youre expecting. I suggest taking a look at the device docs and choose the correct encoding for the connection.
